### ISSUE TITLE
Make the map responsive and viewable on mobile phones

### ISSUE THREAD

**Issue by mateoclarke** [txappleseed/txappleseedmap#25]
_⚠ lower priority says Morgan_

![](http://dsh.re/b3332)

map is blocked by data toggling box

**Comment by mlsintx** [comment]
Yes, this needs help.  I don't have tools for this, though some ideas.

**Comment by codecofee19** [comment]
@mlsintx Have you looked into this? 
https://github.com/chriswhong/cartodb-bootstrap-template

**Comment by mlsintx** [comment]
@codecofee19 and @mateoclarke, I saw this responsive design and it reminded me of another template I have used for some maps, Bootleaf, by Bryan McBride, but it have a lot of other code integrated with it.  It does work really nicely on mobile devices.  I have pulled out some of what we need, in my clumsy way, to use for this project, but don't have a working copy.  If you'd like to look at it tonight, great.  I can work on this and get a map up in the next couple of days, but it won't be tonight.  I did get new geojson up with the districts and demographic info we will need for the popups.

**Comment by mscarey** [comment]
The masthead gets very ugly and the title text disappears when the width of the page decreases. Other than that, the map was technically useable on my phone. I'm sure CSS can solve the problem, but there's a whole lot of existing CSS in the repo, so it's hard to figure out what to change.
